doc/decisions/frontend_keyhandling.md

```diff
@@ -0,0 +1,108 @@
+# Frontend Keybindings
+
+## Problem
+
+In the development of PermaplanT, we have identified the need for a robust keybinding system to enhance the user experience. Keybindings are utilized in various parts of the application, and we need to establish a cohesive approach for their implementation.
+
+## Constraints
+
+Our keybinding solution should adhere to the following constraints:
+
+- **Ease of Integration**: The keybinding system should be straightforward to integrate into different components of the application.
+- **Developer-friendly**: Developers should get an overview where which key binding is assigned.
+- **UX**: Keybindings should not lead to unwanted sideeffects.
+
+## Solution
+
+The following questions have been addressed during the conception:
+
+- **How do we configure keybindings?**
+
+- There should be a single source of truth
+- it should be easy to understand for developers.
+
+- **Where do we add key listeners?**
+
+- While it's easy to add key listeners to focused components like form inputs, it becomes more challenging when dealing with a component like the Konva map.
+- This is because we cannot directly bind keys to the map layer.
```

Comment:
I wouldn't squeeze that in the same decision. It probably doesn't need a decision but just a way that works nicely. (Concept)